posts/2019/05/spyder-4-coming-closer-beta2-is-now-available.md

```diff
@@ -0,0 +1,148 @@
+<!--
+.. title: Spyder 4.0 coming closer, beta2 is now available!
+.. slug: spyder-4-beta2-release
+.. date: 2019-05-16 16:38:50 UTC-05:00
+.. author: Gonzalo Peña-Castellanos
+.. tags: Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+It has been almost two months since I joined Quansight, last April, to start 
+working with [Spyder](https://github.com/spyder-ide/spyder/) maintenance and 
+development. So far, it has been a very exciting and rewarding journey under 
+the guidance of long time Spyder maintainer
+[Carlos Córdoba](https://github.com/ccordoba12).
+This is the first of a series of blog posts we will be writing to showcase 
+updates on the development of Spyder, new planned features and news on the 
+road to Spyder 4.0 and beyond.
+
+Before we describe the features that will be included in the imminent release 
+of Spyder 4.0beta2, I would like to give a warm welcome to
+[Edgar Margffoy](https://github.com/andfoy),
+who recently joined Quansight, and will be working with the Spyder team to
+take its development even further. Edgar is a core Spyder developer and we
+are very excited to have his (almost) full-time commitment to the project.
+
+# Spyder 4.0 beta2 release!
+
+Since August 2018, when the first Beta of the 4.x series was released, the
+Spyder development team has been working really hard on its next release,
+4.0 Beta2. The past couple of months we added the long awaited dark theme
+for the entire interface, we switched our entire code completion and linting
+architecture to the
+[Language Server Protocol](https://microsoft.github.io/language-server-protocol/),
+and merged several other goodies!
+
+have included additional tooling that opens the door to support many other
+programming languages via 
+
+## Dark Theme
+
+This has been a
+[long awaited feature](https://github.com/spyder-ide/spyder/issues/2350)
+and will be the default theme for this new version. Users can still select the
+traditional OS specific theme by going to the **Appearance** section on our
+**Preferences** dialog.
+
+This enhancement was enabled by the work of
+[Colin Duquesnoy](https://github.com/ColinDuquesnoy) and
+[Daniel Pizzeta](https://github.com/DPizzeta)
+and their [QDarkStyle](https://github.com/ColinDuquesnoy/QDarkStyleSheet/)
+package. The Spyder team is now actively collaborating with Colin and Daniel
+to pursue the release of QDarkStyle 3.x, which will be using the
+[qtsass](https://github.com/spyder-ide/qtsass/)
+package to harness the power of SASS/SCSS and allow users to fully customize
+the theme dynamically.
+
+A picture is worth a thousand words, so without further ado, here it is:
+
+![Spyder Dark style](/images/spyder-qdarkstyle.png)
+
+Pretty, right :-) ?
```

Comment:
Could you insert the teaser comment (`<!-- TEASER_END -->`) here?

Reply:
Ahhh that is what it is used for! :-)